lib/vault/api/sys/mount.rb

```diff
@@ -44,8 +49,10 @@ def mounts
     #   the type of mount
     # @param [String] description
     #   a human-friendly description (optional)
-    def mount(path, type, description = nil)
-      payload = { type: type }
+    # @param [String] options
+    # mount type specific options that are passed to the backend (optional)
+    def mount(path, type, description = nil, options = {})
```

Comment:
prefer that options includes `:description` ... maybe some fallback `if options is not a hash, then treat it as description`

Reply:
Not sure I understand this comment, do you mean that options contains a description? Or to remove the extra attribute?

Reply:
both :D 

```ruby
def mount(path, type, options = {})
  description = (options.is_a?(Hash) ? options[:description] : options)
```